diff --git a/engine/build.rs b/engine/build.rs
index f8a856e0..02176bc2 100644
--- a/engine/build.rs
+++ b/engine/build.rs
@@ -1,14 +1,9 @@
-use static_files::NpmBuild;
+use static_files::resource_dir;
+use std::env;
 
 fn main() -> std::io::Result<()> {
     if !cfg!(debug_assertions) && cfg!(feature = "embed_frontend") {
-        NpmBuild::new("../frontend")
-            .install()?
-            .run("generate")?
-            .target("../frontend/.output/public")
-            .change_detection()
-            .to_resource_dir()
-            .build()
+        resource_dir(env!("FRONTEND_DIR")).build()
     } else {
         Ok(())
     }
