diff --git a/src/CMakeModules/Bootstrap_Linux.cmake b/src/CMakeModules/Bootstrap_Linux.cmake
index cae37cc46..fa0c7fbd8 100644
--- a/src/CMakeModules/Bootstrap_Linux.cmake
+++ b/src/CMakeModules/Bootstrap_Linux.cmake
@@ -60,9 +60,6 @@ FIND_PACKAGE (X11 REQUIRED)
 
 if (ENABLE_HTML)
     if (USE_SYSTEM_CEF)
-        set(CEF_LIB_PATH "/usr/lib/casparcg-cef-131")
-        set(CEF_INCLUDE_PATH "/usr/include/casparcg-cef-131")
-
         set(CEF_LIB
             "-Wl,-rpath,${CEF_LIB_PATH} ${CEF_LIB_PATH}/libcef.so"
             "${CEF_LIB_PATH}/libcef_dll_wrapper.a"
