diff --git a/src/modules/decklink/producer/decklink_producer.cpp b/src/modules/decklink/producer/decklink_producer.cpp
index f42b5ea9c..91fbd9a90 100644
--- a/src/modules/decklink/producer/decklink_producer.cpp
+++ b/src/modules/decklink/producer/decklink_producer.cpp
@@ -637,7 +637,6 @@ class decklink_producer : public IDeckLinkInputCallback
                         return S_OK;
                     }
 
-                    audio_filter_.sink->inputs[0]->min_samples = audio_cadence_[0];
                     if (av_buffersink_get_frame_flags(audio_filter_.sink, av_audio.get(), AV_BUFFERSINK_FLAG_PEEK) <
                         0) {
                         return S_OK;
